lib/cb/clients/recommendation.rb

```diff
@@ -65,12 +65,15 @@ def self.for_user(*args)
       def self.for_company(company_did)
         my_api = Cb::Utils::Api.instance
         json_hash = my_api.cb_get(Cb.configuration.uri_recommendation_for_company, query: { CompanyDID: company_did })
-
         jobs = []
         if json_hash.key?('Results')
           if json_hash['Results'].key?('JobRecommendation')
-            json_hash['Results']['JobRecommendation']['Jobs'].each do |cur_job|
-              jobs << Models::Job.new(cur_job)
+            if json_hash['Results']['JobRecommendation'].key?('Jobs')
+              if json_hash['Results']['JobRecommendation']['Jobs'].key?('CompanyJob')
+                json_hash['Results']['JobRecommendation']['Jobs']['CompanyJob'].each do |cur_job|
+                  jobs << Models::Job.new(cur_job)
+                end
+              end
             end
```

Comment:
So if the company did isn't valid, or has no other jobs, it will return blank. I guess that would be ok. 

Does it hurt to continue evaluating if 'Results' isnt there? What I mean, is will it look for 'Results', return {} if it's not there, and then look for 'JobRecommendation' and so on?